api/porch/register.go

```diff
@@ -38,8 +38,8 @@ var (
 
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
-		&Package{},
-		&PackageList{},
+		&PorchPackage{},
```

Comment:
Curious as to why the renaming is needed? Thanks

Reply:
It uses [kube-codegen](https://github.com/kubernetes/code-generator/tree/master) to generate the api code.

With naming the CR Package, the code generator didn't like that apparently as package is a keyword in golang, so they forked it and customised it [here](https://github.com/platkrm/code-generator). Didn't dig into what they did really.

As part of the migration/clean up plan, the Package and Function CRs will be removed, so the idea here is to provide a stop gap solution (hopefully without breaking anything), by renaming to "PorchPackage.."

Reply:
> Curious as to why the renaming is needed? Thanks

@vjayaramrh Are you OK with this?

Reply:
THanks for the response, I am good.